### ISSUE TITLE
Import online recipes by parsing recipe schema

### ISSUE THREAD

**Issue by spencerpogo** [cooklang/cookcli#15]
To get their recipes in search engine's rich recipe search results, many recipe sites describe their recipes with a JSON schema in a special HTML tag. Some recipe apps can interpret this schema in order to import recipes. CookCLI could parse this schema and generate a cook file that could be reviewed by a human. 

More information about structured data and recipe schemas is available at [Google developers](https://developers.google.com/search/docs/advanced/structured-data/intro-structured-data).

**Comment by dubadub** [comment]
@Scoder12 It's a great idea and many people were mentioning it. I'm wondering how you see it can be integrated with the CLI? Something like fetching recipe from the web-page and creating file locally?

**Comment by Scoder12** [comment]
Yes, that's what I was thinking. There could be a CLI command to fetch a
URL a generate a cook file from it. I'd make a PR but I'd have to learn
swift first.

**Comment by dubadub** [comment]
Resolved in https://github.com/cooklang/cookcli/pull/142
